src/elf/program_header.rs

```diff
@@ -308,26 +310,35 @@ macro_rules! elf_program_header_std_impl { ($size:ty) => {
 
         impl ProgramHeader {
             #[cfg(feature = "endian_fd")]
-            pub fn parse(bytes: &[u8], mut offset: usize, count: usize, ctx: ::scroll::Endian) -> Result<Vec<ProgramHeader>> {
+            pub fn parse(
+                bytes: &[u8],
+                mut offset: usize,
+                count: usize,
+                ctx: ::scroll::Endian,
+            ) -> Result<Vec<ProgramHeader>> {
                 use scroll::Pread;
                 let mut program_headers = vec![ProgramHeader::default(); count];
                 let offset = &mut offset;
                 bytes.gread_inout_with(offset, &mut program_headers, ctx)?;
                 Ok(program_headers)
             }
 
+            #[cfg(feature = "alloc")]
             pub fn from_bytes(bytes: &[u8], phnum: usize) -> Vec<ProgramHeader> {
                 let mut phdrs = vec![ProgramHeader::default(); phnum];
-                phdrs.copy_from_bytes(bytes).expect("buffer is too short for given number of entries");
+                phdrs
+                    .copy_from_bytes(bytes)
+                    .expect("buffer is too short for given number of entries");
                 phdrs
             }
 
             /// # Safety
             ///
             /// This function creates a `ProgramHeader` directly from a raw pointer
-            pub unsafe fn from_raw_parts<'a>(phdrp: *const ProgramHeader,
-                                             phnum: usize)
-                                             -> &'a [ProgramHeader] {
+            pub unsafe fn from_raw_parts<'a>(
```

Comment:
ditto, same; i wonder did newer versions of formatter alter it's line breaking/indentation rules

Reply:
...or change with respect to how formatting worked inside of macro expansions, perhaps?